jedi/api/classes.py

```diff
@@ -734,7 +738,7 @@ def desc_with_module(self):
             DeprecationWarning,
             stacklevel=2
         )
-        position = '' if self.in_builtin_module else '@%s' % self.line
+        position = '' if self.in_builtin_module() else '@%s' % self.line
```

Comment:
This mistake was made in 0cb23dcfa2e54481c3d96c9fb281641a89a86cd4. That's from 2014. So IMO we should probably just remove the position. It was never applied anyway and nobody asked for it. Just remove this whole line and we should be good :)

Reply:
Aha! It was referenced from emacs-jedi, but with that observation it is completely replaced by full_name.